test/v1_0_3/non_templating.js

```diff
@@ -4294,7 +4290,7 @@
           .expect(400)
           .end()
           .get(helper.getEndpointStatements() + '?statementId=' + correct.id)
-          .wait(genDelay(stmtTime, '/statmentId=' + correct.id, correct.id))
+          .wait(genDelay(stmtTime, '?statmentId=' + correct.id, correct.id))
```

Comment:
Absolutely, 'statement', thanks Freddie  (con-thru #56 and 5c99765)

On Thu, Sep 1, 2016 at 9:48 PM, oconnetf <notifications@github.com> wrote:

> In test/v1_0_3/non_templating.js
> <https://github.com/adlnet/lrs-conformance-test-suite/pull/54#discussion_r77282951>
> :
>
> > @@ -4294,7 +4290,7 @@
> >            .expect(400)
> >            .end()
> >            .get(helper.getEndpointStatements() + '?statementId=' + correct.id)
> > -          .wait(genDelay(stmtTime, '/statmentId=' + correct.id, correct.id))
> > +          .wait(genDelay(stmtTime, '?statmentId=' + correct.id, correct.id))
>
> This is just a guess, but shouldn't that read "statementId" and not "
> statmentId"?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/adlnet/lrs-conformance-test-suite/pull/54/files/e24538207d98961e716fd2e625edae7e451a4d6e#r77282951>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF5rA0NopRwM7u-Wd7y5p4iLRIiTu7J_ks5ql4B7gaJpZM4JzG_O>
> .
>